### ISSUE TITLE
all of the objects have one companion object in ram ?!!?!

### ISSUE THREAD

**Issue by amirhosenebrahimi** [ansman/kotshi#142]
all of the objects have one companion object in ram ?!!?! 
do not kill gorebridge collectors !

**Comment by ansman** [comment]
Yes, the adapters have a companion object but since the fields will be static they are empty and can be removed by Proguard or r8 on Android. Regardless they are not likely to cause any issues.

The reason why the options are static is so they can be reused if the same adapter is used multiple times.

**Comment by ansman** [comment]
Regardless, I did end up removing the companion objects in #143

**Comment by amirhosenebrahimi** [comment]
thank you :-)
